tasks/configure-Debian.yml

```diff
@@ -38,6 +38,23 @@
   notify: restart apache
   when: apache_create_vhosts
 
+- name: Configure Apache (Add listen port for every vhost).
+  blockinfile:
+    dest: "{{ apache_server_root }}/ports.conf"
```

Comment:
I just looking at the code. May be I could refactor at the same time this part and another part which have the same pattern (at the begining of every _configure-${OS}.yml_ file) :

```
- name: Configure Apache.
  lineinfile:
    dest: "{{ apache_server_root }}/ports.conf"
...
```
What do you thing about this ?